modules/basegl/processors/mesh2drenderprocessorgl.cpp

```diff
@@ -84,16 +84,16 @@ Mesh2DRenderProcessorGL::Mesh2DRenderProcessorGL()
     addProperty(bottom_);
     addProperty(top_);
 
-    inport_.onChange([this]() { updateDrawers(); });
+    inport_.onChange(this, &Mesh2DRenderProcessorGL::updateDrawers);
```

Comment:
using this calling convention is deprecated! `[this]() { updateDrawers(); }` should be used. 
Please undo.

Reply:
Fixed